### DISCUSSION TITLE
Porting Default Debian 12.8 Cinnamon to Spiral Linux Cinnamon

### DISCUSSION THREAD

**Discussion by carlosEgarc** [SpiralLinux/SpiralLinux-project#509] [General]
Hello. Any way I can port default Debian 12.8 to Spiral Linux ? 

Hola te escribo en español mas fácil para mi y se que tu hablas español también. Ya el computador de mi casa lo estoy usando con Spiral Linux Cinnamon, Pero en mi oficina tengo uno con Debian 12.8 y me gustaría poder llevarlo a Spiral Linux  sin tener que re instalar. No se si es posible "jugando" con los repositorios se pueda lograr hacer ese porting.  Al momento el único repositorio adicional que tengo instalado es el de Google Chrome. 

Gracias

Carlos Garcia

**Comment by geckolinux** [comment]
Hola amigo un gusto saludarte.

> No se si es posible "jugando" con los repositorios se pueda lograr hacer ese porting

En realidad no. Un concepto fundamental que se debe entender con SpiralLinux es que no es una "distro" con paquetes DEB y repositorios propios. Esa decisión se tomó para que SpiralLinux no dependa de mí como desarrollador, sino de los recursos técnicos y humanos mucho más abundantes del proyecto Debian. Así que no existe ningún mecanismo que permita aplicar un conjunto de configuraciones de SpiralLinux a un sistema ya instalado. La configuración que viene por defecto con SpiraLinux se aplica durante la etapa de la creación de la imagen ISO y después de eso el sistema no mantiene ninguna conexión con el proyecto SpiralLinux. Para aplicar esa configuración a un sistema ya instalado habría que copiar un montón de archivos manualmente de los archivos `.tar.gz` que subo a Github y también instalar algunos paquetes de la lista que utilizo para generar las imágenes ISO de SpiralLinux.

> tengo uno con Debian 12.8 y me gustaría poder llevarlo a Spiral Linux sin tener que re instalar

Todo es posible, pero realmente es una cuestión de si vale la pena hacerlo. Y dado lo fácil que es instalar SpiralLinux desde cero y lo trabajoso que sería pasar manualmente todas las configuraciones de SpiralLinux a tu instalación de Debian, me inclinaría a decir que no valdría la pena.

Lo que te podría sugerir tal vez sería que hagas un respaldo de tu carpeta actual de `/home/`  y luego formatear el disco con SpiralLinux y finalmente volver a cargar los archivos de tu copia de respaldo.

**Comment by carlosEgarc** [comment]
Hola, muy buenos días, Gracias por tu tiempo y amplia respuesta. Definitivamente si seria hacer una copia de soporte de HOME e instalar Spiral desde cero. En todo caso no debería ser un proceso complejo toda ves que solo son archivos de LibreOffice y unos tantos pdf que no son mas de 10 Gigas.  

Gracias por tu excelente trabajo y aporte.

Carlos Garcia (Colombia) :)
